docs/releases/v0.1.x/v0.1.0-beta1.mdx

```diff
@@ -1,4 +1,5 @@
 import { Meta } from '@storybook/blocks';
+import Code from '../../components/Code'
```

Comment:
Q: why we need to use this component? Why we can't keep use markdown

Reply:
This is how it looks with `<Code />`
<img width="797" alt="image" src="https://github.com/infermedica/component-library/assets/37744371/2d99c122-56f8-4f43-8f48-bad73a4676d9">
and this is how it looks as a `"```js....```"`
<img width="355" alt="image" src="https://github.com/infermedica/component-library/assets/37744371/c0fcae79-0ad3-4dcd-af5d-afcb3bf73703">